clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -127,6 +127,9 @@ public Fetcher(ConsumerNetworkClient client,
         this.retryBackoffMs = retryBackoffMs;
     }
 
+    public int getMaxPollRecords() {
```

Comment:
Nit: This should be called `maxPollRecords` as we don't use the `get` prefix for methods like this.